src/app/view/ChessBoardView.ts

```diff
@@ -19,3 +20,47 @@ export interface ChessBoardView {
   //showAvailableMoves
   //showSelectedPiece
 }
+*/
+
+export class Square {
+  constructor(private position_x: number, private position_y: number, private color: SquareColor) {}
+
+  setHtmlElement(): HTMLElement {
+    const position = this.mapSquarePosition();
+    const squareElement = document.createElement('div');
+    squareElement.id = position;
+    squareElement.classList.add(`square--${this.color}`);
+    return squareElement;
+  }
+
+  mapSquarePosition(): string {
+    const columns = 'abcdefgh'.split('').reverse();
+    const rows = '12345678'.split('');
+    return `${columns[this.position_x]}${rows[this.position_y]}`;
+  }
+}
+
+export type SquareColor = 'light' | 'dark';
+export class Chessboard {
+  constructor(private id: string = 'chessboard', private className: string = 'chessboard') {}
+
+  private setHtmlElement(): HTMLElement {
+    const boardElement = document.createElement('div');
+    boardElement.id = this.id;
+    boardElement.classList.add(this.className);
+    return boardElement;
+  }
+
+  createBoard(): HTMLElement {
+    const boardHtml = this.setHtmlElement();
+    for (let y = 7; y >= 0; y--) {
+      for (let x = 7; x >= 0; x--) {
```

Comment:
1) co to jest "x" i "y" - przydałyby się zmienić nazwy na opisowe bardziej
2) Jeśli chcesz wielkość szachowwnicy to jest teś w pliku z "const'ami" 9pobierz develop jesli nie masz)

Reply:
Ad.1 Jak chcecie :) X i Y też mogą być w przypadku układu współrzędnych. Jedyne co to akurat tutaj nie podoba mi się liczenie od 0. Nie wiem czy to nie zrodzi problemów, bo szachownica jednak ma pola od 1. 
Ad.2 to co pisałem wcześniej - te const są do modelu. Tutaj nie bazujmy na tym.

Reply:
Dziękuję jeszcze raz. Masz racje, że CONSTY to MODEL a nie VIEW.
Nie skojarzyłem. Czyli w takim razie i to też cofam

Reply:
Liczenie jest do 0, tylko dlatego, że potem jest to indeks do tablicy. Jest to mapowane na odpowiednią notację pola w klasie Square.